pilot/pkg/xds/fake.go

```diff
@@ -47,10 +47,10 @@ import (
 )
 
 type FakeOptions struct {
-	// If provided, these objects will be used directly
-	KubernetesObjects []runtime.Object
-	// If provided, the yaml string will be parsed and used as objects
-	KubernetesObjectString string
+	// If provided, each map-item will use the key as the cluster id for a registry containing the given objects
```

Comment:
nit: can we make these a dedicated KubernetesObjectsByObject field or something and keep the originals? This leaks multicluster details when majority of tests do not need to specify this

Reply:
Done